core/templates/services/contextual/url.service.ts

```diff
@@ -61,7 +61,7 @@ export class UrlService {
   So exact type of this function can not be determined
   https://github.com/oppia/oppia/pull/7834#issuecomment-547896982 */
   getUrlParams(): UrlParamsType {
-    let params = {};
+    let params: Record<string, string> = {};
```

Comment:
Please define an interface and have a name for the key and value. (Ditto here and elsewhere)

Reply:
Done! I saw that there was an interface that I could reuse in that same file and I used it.